src/js/controller/instream-html5.js

```diff
@@ -145,6 +145,10 @@ define([
                 provider.mute(_model.get('mute'));
 
                 _adModel.on('change:state', changeStateEvent, _this);
+            } else if(provider.state === 'playing') {
```

Comment:
@egreaves here's the tech-debt ticket https://github.com/jwplayer/jwplayer/issues/896